package.json

```diff
@@ -2,6 +2,7 @@
   "name": "@enact/docs-utils",
   "description": "Utilities for parsing and validating JSDoc in Enact projects",
   "version": "0.3.7",
+  "type": "module",
```

Comment:
@adrian-cocoara-lgp 

> Extension for .eslintrc had to be changed to .cjs in order to fix this travis build: https://app.travis-ci.com/github/enactjs/docs-utils/jobs/585383436

I think this line caused this travis fail.

Reply:
Okay, thanks